.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Please follow these steps to help us fixing the bug:
+
+1. Make sure it is a bug for this repo
+  * Can you reproduce the bug with Scala/JVM? If yes, report it to https://github.com/scala/bug instead
+  * Make sure it is not one of the intended semantic differences: https://www.scala-js.org/doc/semantics.html
+  * Does the bug involve macros? If yes, make sure to reproduce *without* macros, or file an issue to the relevant macro library instead
```

Comment:
This point is a bit unclear to me: What if it involves a macro the user has written themselves?